source/opt/instruction.h

```diff
@@ -821,16 +819,16 @@ inline bool Instruction::WhileEachInOperand(
 
 inline void Instruction::ForEachInOperand(
     const std::function<void(uint32_t*)>& f) {
-  WhileEachInOperand([&f](uint32_t* op) {
-    f(op);
+  WhileEachInOperand([&f](uint32_t* operand_id) {
+    f(operand_id);
```

Comment:
The argument is not necessarily an id.  It could be a storage class, a literal, or various other things.

Reply:
Done.